### PR TITLE
New types

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
With this one, not that much types are remaining. :D

cc @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
Manual part don't needed, Gir can handle it.
```
[[object.function.parameter]]
    name = "region"
    const = true
```

**Comment by GuillaumeGomez** [comment]
Updated.

File: Gir.toml [hunk 1]
```diff
@@ -1753,6 +1754,17 @@ status = "generate"
...
+    name = "input_shape_combine_region"
+        [[object.function.parameter]]
+        name = "region"
+	const = true
+    [[object.function]]
+    name = "shape_combine_region"
+        [[object.function.parameter]]
+        name = "region"
+	const = true
+
```
Comment (EPashkin) [inline Gir.toml]: Extra line and TAB instead SPACEs in previous line

Comment (EPashkin) [inline Gir.toml]: *lines

**Comment by EPashkin** [comment]
Other that small nit LGFM. Thanks

File: Gir.toml (hunk 1 above)
Comment (GuillaumeGomez) [inline Gir.toml]: Damn emacs.

**Comment by GuillaumeGomez** [comment]
Just waiting for CIs (just in case...).

File: Gir.toml [hunk 2]
```diff
@@ -1753,6 +1754,17 @@ status = "generate"
...
+    name = "input_shape_combine_region"
+        [[object.function.parameter]]
+        name = "region"
+        const = true
+    [[object.function]]
+    name = "shape_combine_region"
+        [[object.function.parameter]]
+        name = "region"
+        const = true
+
```
Comment (EPashkin) [inline Gir.toml]: Extra line still here ;)

Comment (GuillaumeGomez) [inline Gir.toml]: Sorry!

**Comment by GuillaumeGomez** [comment]
Ok, removed the empty line. Just merging since it seems to be ok. Thanks for the review @EPashkin! :)

**Pull request merged**